Makefile

```diff
@@ -532,6 +532,11 @@ flutter.run:
 flutter.web.assets:
 	@rm -rf flutter/assets/pkg
 	wasm-pack build -d flutter/assets/pkg --no-typescript -t web
+		
+	sed -i "s/medea_jason_bg.wasm/medea_jason_bg.wasm?\
```

Comment:
This doesn't work, `sha256` invocation does not wait for build to finish for some reason:
```
$ make flutter.web.assets
sha256sum: flutter/assets/pkg/medea_jason_bg.wasm: No such file or directory
wasm-pack build -d flutter/assets/pkg --no-typescript -t web
[INFO]: Checking for the Wasm target...
[INFO]: Compiling to Wasm...
    Finished release [optimized] target(s) in 0.09s
``` 

So the resulting `medea_jason.js` looks like this:

```js
async function init(input) {
    if (typeof input === 'undefined') {
        input = new URL('medea_jason_bg.wasm?', import.meta.url);
    }
...
```